src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -320,6 +319,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_OVER IMPLEMENTATION.
         lv_favorite_icon = 'star/blue'.
         lv_favorite_class = 'favorite'.
       ELSE.
+        lv_suppress_title = abap_true.
```

Comment:
from a user perspective, I think this can easily be considered a bug, some places titles are shown and some not

Suggest buffering the usernames

for the package description TDEVCT is fully buffered on the application server, so it should be very fast

Reply:
really you open the non-favorites only if you want to favorite a repo, and then the alt text will be shown

Reply:
![image](https://user-images.githubusercontent.com/5097067/108596111-36aab800-7383-11eb-804a-4163170f2dac.png)
tdevct is actually the most problematic, there are not going to be many usernames

Reply:
also `susr_user_address_read` already contains buffering logic

Reply:
okay, then what about always skipping the titles on the overview page for everything? that would also give a consistent/more transparent experience

Reply:
alternatively, linking users and packages to 2 new pages, which can show the full information regarding the user and packages

^new pages, so it works in a possible future web edition

Reply:
Now that you mention it, the package is already a link to the package object and clicking a username shows a popup with the complete user information. I no longer see any point in the descriptions at all. @mbtools any strong feelings about this?